crates/polars-utils/src/pl_str.rs

```diff
@@ -0,0 +1,124 @@
+use compact_str::CompactString;
```

Comment:
I think I mentioned `compact_str` at some point so I am to blame, but I did notice it has `O(n)` clone so this isn't the right choice for us.

Reply:
All good - I was also aware of this - I was just doing some experimentation with `const fn` here 👍